.changeset/thirty-geckos-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
```suggestion
'astro': minor
```

Reply:
Do I need to change jsdoc for new config option? It has '@version' item

Reply:
Ah yes! Should be `@version 3.1.0`